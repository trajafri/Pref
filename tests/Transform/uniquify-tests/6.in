(lambda (x) (lambda (y) (lambda (x) (lambda (y) (+ x y)))))

(let ((x 1)) (lambda (y) (lambda (x) (let ((y 2)) (+ x y)))))
