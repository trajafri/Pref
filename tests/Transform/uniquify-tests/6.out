(lambda (x.0) (lambda (y.0) (lambda (x.1) (lambda (y.1) (+ x.1 y.1)))))

(let ((x.2 1)) (lambda (y.2) (lambda (x.3) (let ((y.3 2)) (+ x.3 y.3)))))
